src/libraries/System.Reflection.Emit/tests/PersistableAssemblyBuilder/AssemblySaveAssemblyBuilder.cs

```diff
@@ -213,10 +213,12 @@ public void AssemblyWithDifferentTypes()
                 eventb.SetRemoveOnMethod(mbRemove);
                 tbEvents.CreateType();
 
-                saveMethod.Invoke(ab, [file.Path]);
+                ab.Save(file.Path);
 
-                Assembly assemblyFromDisk = AssemblySaveTools.LoadAssemblyFromPath(file.Path);
```

Comment:
Is AssemblySaveTools.LoadAssemblyFromPath() used elsewhere? Can it be removed?

Reply:
No, it is removed